examples/MomentoFSharpApplication/Program.fs

```diff
@@ -30,5 +35,5 @@ let exerciseCache() = (
 
 [<EntryPoint>]
 let main(argv :string[]) =
-    exerciseCache()
+    let _ = exerciseCache()
```

Comment:
Is `_` similar to Go? (not using the assigned variable?)

Reply:
correct.  But f# really wants you to avoid statements in favor of expressions so it was giving me a warning that I wasn't doing anything with the return value.